pyvista/core/pointset.py

```diff
@@ -3374,7 +3374,7 @@ def cell_coords(
         else:
             if isinstance(coords[0], np.ndarray):
                 return np.stack(coords, axis=1)
-            return np.asanyarray(coords)
+            return np.asanyarray(coords)  # type: ignore[unreachable]
```

Comment:
This logic seems odd. `https://numpy.org/doc/stable/reference/generated/numpy.unravel_index.html` seems to always return a tuple of arrays. Perhaps what was intended was to check if the array was scalar or not?
``` diff
- if isinstance(coords[0], np.ndarray):
+ if coords[0].ndim > 0:
```
To check if it's a scalar or array (and stack it if it is an array)

Reply:
This is also ok to fix later